docs/niscope/enums.rst

```diff
@@ -557,7 +557,7 @@ Enums used in NI-SCOPE
 
 .. py:data:: FlexFIRAntialiasFilterType
 
-    .. py:attribute:: FlexFIRAntialiasFilterType._48_TAP_STANDARD
+    .. py:attribute:: FlexFIRAntialiasFilterType.FOURTYEIGHT_TAP_STANDARD
```

Comment:
Looked at NI-SCOPE .NET API. Names are: Tap48Standard , Tap16Hanning and so forth. I think that if we cannot base the Python names directly from the C names, then we should fall back to basing them off the .NET API rather than invent another transform.

This would be, then, `tap_48_standard`.

See http://digital.ni.com/manuals.nsf/websearch/9F213B8EDEB716B986257EBA0077C1FA

Reply:
Does that mean we also want to follow their lead on the video triggering enum?

`NISCOPE_VAL_1080I_50_FIELDS_PER_SECOND` == `I1080Lines50FieldsPerSecond`

I don't really like moving the 'P' and 'I' to the beginning, because it is not `I1080` video, it is `1080I` video. Also it isn't a tap 48 filter, it is a 48 tap filter. To me, this is not the right solution to this issue, since this changes the familiar name of these things.

Reply:
Ok. So let it be recorded then in this PR that we're making a conscious decision to ignore the precedent in naming set by .NET.